src/ooxml/java/org/apache/poi/xddf/usermodel/chart/XDDFChart.java

```diff
@@ -81,6 +82,27 @@
 
 @Beta
 public abstract class XDDFChart extends POIXMLDocumentPart implements TextContainer {
+    
```

Comment:
moved constant variable code from XWPFChart TO XDDFChart

Reply:
with the moved constants, we can't delete them from XWPFChart as it would cause issues for other users but could you set the XWPFChart values to be be based on the XDDF ones?

```
public static final int DEFAULT_WIDTH = XDDFChart.DEFAULT_WIDTH; 
```

Reply:
merged required code.